compiler/IREmitter/sends.cc

```diff
@@ -151,7 +151,8 @@ InlinedVector<ApplicableIntrinsic, NUM_INTRINSICS> applicableIntrinsics(MethodCa
 
         auto potentialClasses = symbolBasedIntrinsic->applicableClasses(mcctx.cs);
         for (auto &c : potentialClasses) {
-            auto leftType = core::Types::dropSubtypesOf(mcctx.cs, remainingType, c);
+            InlinedVector<ClassOrModuleRef, 1> toDrop{c};
```

Comment:
```suggestion
            auto toDrop = absl::make_span(&c, 1);
```

or something like that?